docs/tutorials/fundamentals/installation.md

```diff
@@ -116,8 +116,16 @@ If you want to install napari with PySide2 as the backend you need to install it
 
 ```sh
 conda install -c conda-forge "napari=*=*pyside2"
-```  
+``` 
+````
 
+````{note}
+It can happend that `conda`'s build-in solver is not able to solve all denpendencies. This can happen even in a clean environment
+depending on your operating system. In this case use `pip` for the installation or if that is no option, install `mamba` and use 
```

Comment:
```suggestion
depending on your operating system. In this case, you can try to make the solvers job easier by specifying the version of napari, using, for example, `conda install  -c conda-forge napari=0.4.17` or by first installing a specific Qt backend, for example `conda install -c conda-forge pyside2` and then installing napari. Alternately, you can install [`mamba`](https://github.com/mamba-org/mamba) which has a newer (and faster) solver and use 
```

Reply:
I'm not sure I would encourage mixing in `pip` here.
I also added some tips that @jaimergp posted previously about making the solver job easier.

Reply:
@melissawm 
Giving the example of installing a napari version, is it possible to get the current stable napari version as a variable, so the docs are always up to date without having to remember to update this number?

Reply:
Definitely, good friends don't let you mix pip and conda unless absolutely necessary.

Reply:
I would simplify this note further. Something like:

> In some cases, conda's default solver will struggle to find out which packages need to be installed for `napari`. If it takes too long or you get the wrong version (see below), consider:
>
> - Overriding your default channels to use only conda-forge, and specifying the napari and Python versions explicitly`conda create -n napari -c conda-forge --override-channels python=3.10 napari=<latest>`
> - If that's not enough, you can try installing `mamba` in your `base` environment with `conda install -n base -c conda-forge mamba`, and use its faster solver to run the above instructions.

Reply:
Love it!

Reply:
Sorry for the typos >.<

> Definitely, good friends don't let you mix pip and conda unless absolutely necessary.

Sure, but the idea was just to use conda for the environment and pip for everything else instead. I guess that part was not clear enough ^^

> I would simplify this note further. Something like:
> 
> > In some cases, conda's default solver will struggle to find out which packages need to be installed for `napari`. If it takes too long or you get the wrong version (see below), consider:
> > 
> > * Overriding your default channels to use only conda-forge, and specifying the napari and Python versions explicitly`conda create -n napari -c conda-forge --override-channels python=3.10 napari=<latest>`
> > * If that's not enough, you can try installing `mamba` in your `base` environment with `conda install -n base -c conda-forge mamba`, and use its faster solver to run the above instructions.

This looks really good.

Reply:
Yea was just a if it does not work try the above xD well kinda redundant I know.

Reply:
> @melissawm Giving the example of installing a napari version, is it possible to get the current stable napari version as a variable, so the docs are always up to date without having to remember to update this number?

Yes - if you have this `stable_version` variable defined in a `myst_substitutions` dictionary in the conf.py file, you can print it anywhere in the docs using the following syntax:

```
{{ stable_version }}
```

See [this commit](https://github.com/melissawm/napari-docs/commit/8443859f488bd37b6989f86292fefaca02498f3e) for an example.